build.gradle

```diff
@@ -21,17 +21,17 @@ allprojects {
     apply plugin: 'idea'
     apply plugin: 'maven-publish'
 
-    sourceCompatibility = 1.7
```

Comment:
Apparently you are using features from java 8 (default methods) so this need to be updated accordingly 
![image](https://github.com/kevin-wayne/algs4/assets/3080529/a2e0a4c4-aa51-4f91-9dbf-ff47a196fda6)